openstudiocore/CMake/FindEnergyPlus.cmake

```diff
@@ -50,9 +50,9 @@ else()
   file(GLOB ENERGYPLUS_POSSIBLE_PATHS "/usr/local/[eE]nergy[pP]lus?*")
 endif()
 
-# sort possible paths in reverse order
-list(APPEND ENERGYPLUS_POSSIBLE_PATHS $ENV{ENERGYPLUSDIR})
+# sort possible paths in reverse order, favoring the ENERGYPLUSDIR environment variable
```

Comment:
@axelstudios the second use case is that you want to build on Peregrine and there is no package available for you to point to so you have to build E+ (and now Radiance) yourself.  So the SHA would be the same, just different build.